src/scss/elements/_timestamp.scss

```diff
@@ -16,71 +16,55 @@
 @mixin oTeaserTimestampVariants {
 	@keyframes live-blog-pulse {
 		0% {
-			transform: scale(1.5);
+			transform: scale(1);
 		}
 		50% {
-			transform: scale(1);
+			transform: scale(1.5);
 		}
 		100% {
-			transform: scale(1.5);
+			transform: scale(1);
 		}
 	}
 
 	.o-teaser__timestamp--new,
-	.o-teaser__timestamp--updated,
-	.o-teaser__timestamp--inprogress {
+	.o-teaser__timestamp--updated {
```

Comment:
I've separated `--inprogress`  so it can use the specific palette color for live blogs.